<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-GS8283A" ignored="0" name="Personal Loan" tests="6" failures="1" timestamp="25 Apr 2019 02:07:20 GMT" time="0.066" errors="0">
  <testcase name="loginAPIHomeLoan" time="0.008" classname="test.day4"/>
  <testcase name="Demo" time="0.011" classname="test.day1"/>
  <testcase name="pLoan" time="0.012" classname="test.day2"/>
  <testcase name="demo2" time="0.006" classname="test.day1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.day1.demo2(day1.java:28)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- demo2 -->
  <testcase name="mobileLoginHomeLoan" time="0.002" classname="test.day4"/>
  <testcase name="webLoginHomeLoan" time="0.004" classname="test.day4"/>
</testsuite> <!-- Personal Loan -->
